open-metadata-implementation/access-services/subject-area/subject-area-client/src/main/java/org/odpi/openmetadata/accessservices/subjectarea/client/relationships/SubjectAreaLine.java

```diff
@@ -384,6 +293,6 @@ protected SubjectAreaProjectScopeClient(SubjectAreaRestClient subjectAreaRestCli
         if (cache.containsKey(clazz)) {
             return (SubjectAreaRelationshipClient<T>) cache.get(clazz);
         }
-        return null;
+        throw new IllegalArgumentException("Not found client for " + clazz.getName());
```

Comment:
please use the standard way of doing messages for subject area -add a new entry into SubjectAreaErrorCode and refer to that